### ISSUE TITLE
Dopamine prevents Windows from sleeping when a song is paused

### ISSUE THREAD

**Issue by marcosbozzani** [digimezzo/dopamine-windows#1003]
## I'm submitting a...

<pre><code>
[ ] Regression 
[x] Bug report  
[x] Feature request
</code></pre>

## I'm using these versions

* Windows version: 1903 (OS Build 18362.175)
* Dopamine version: 1.5.14 (Release)

## Reproduction steps

* Step 1: Open Dopamine
* Step 2: Start a song
* Step 3: Pause the song
* Step 4: Open Powershell as admin
* Step 5: Run `powercfg -requests`

## Observed behavior

```
PS C:\Windows\system32> powercfg -requests
DISPLAY:
None.

SYSTEM:
[DRIVER] High Definition Audio Device (HDAUDIO\FUNC_01&VEN_14F1&DEV_506E&SUBSYS_10280572&REV_1000\4&b8a6c2&0&0001)
An audio stream is currently in use.

AWAYMODE:
None.

EXECUTION:
None.

PERFBOOST:
None.

ACTIVELOCKSCREEN:
None.
```

## Desired behavior

```
PS C:\Windows\system32> powercfg -requests
DISPLAY:
None.

SYSTEM:
None.

AWAYMODE:
None.

EXECUTION:
None.

PERFBOOST:
None.

ACTIVELOCKSCREEN:
None.
```

## Additional information

The stay awake request only goes away when the last song is finished in the playlist or when Dopamine is closed. When I pause a media in other media players installed in my system (e.g. mpv.net), the stay awake request is canceled and only returns when the media is played again.

Could Dopamine be changed to release the stay awake request when it is paused?

If this is the intended behavior, could Dopamine offer a switch in settings the allows it to reproduce the other players behavior?

Thanks! 😄

**Comment by digimezzo** [comment]
Thanks for reporting this. This is not the intended behaviour. I'll look into it.

**Comment by digimezzo** [comment]
Fixed in efaf4e9f76fcad321e862e4b28cb6e02b39705f6
